api/server/router/network/backend.go

```diff
@@ -0,0 +1,22 @@
+package network
+
+import (
+	// network config needs to be pull out to here
+	"github.com/docker/docker/daemon/network"
+
+	"github.com/docker/libnetwork"
+)
+
+// Backend is all the methods that need to be implemented to provide
```

Comment:
Well `Backend` is the name of the struct so it looks fine from the `golint` point of view :sweat_smile: